src/modules/item/export.ts

```diff
@@ -96,6 +100,10 @@ export async function calculateFinalSize(item: Item, newContents: Record<string,
   return imageSize + calculateFilesSize(uniqueFiles)
 }
 
+export function calculateThumbnailFinalSize(thumbnail: Blob): number {
```

Comment:
As the calculating the thumbnail size implies calculating a file size, I would rename this function to `calculateFileZie` to compute the size of a single file or use it as is. What do you think?

Reply:
+1